t/43-dbtest.t

```diff
@@ -5,7 +5,7 @@ if (!defined $ENV{LSMB_TEST_DB}){
         plan skip_all => 'Skipping all.  Told not to test db.';
 }
 else {
-        plan tests => 470;
+        plan tests => 465;
```

Comment:
I like the random form_id keys.

Session ids use a random number to generate the token.  The sequential portion is just for foreign key purposes.  I think changing that is not worth the possibility of accidental collision.